src/modules/ce/components/ReferralStatusChip.tsx

```diff
@@ -8,35 +8,43 @@ interface Props {
 }
 const ReferralStatusChip: React.FC<Props> = ({ status }) => {
   const baseChipSx: SxProps = {
-    borderWeight: '1px',
-    borderColor: 'borders.main',
-    borderStyle: 'solid',
     fontWeight: 600,
+    fontSize: '14px',
+    backgroundColor: 'grayscale.surface',
```

Comment:
likewise lets either move styles to theme, or leave them with the default style. specifying font size in pixels here is not ideal.